src/framework/ui/dropdown/dropdown.component.tsx

```diff
@@ -115,8 +115,11 @@ class DropdownComponent extends React.Component<DropdownProps, State> {
     this.setState({ visible });
   };
 
-  private onPress = () => {
+  private onPress = (event: GestureResponderEvent) => {
     this.props.dispatch([]);
+    if (this.props.onPress) {
```

Comment:
Do we have any expected behavior when we pass `onPress` property?

Reply:
I don't really keep it in mind. Just for test coverage